src/components/pages/HomePage.jsx

```diff
@@ -1,28 +1,114 @@
-import { useNavigate } from 'react-router-dom';
+import React from 'react';
+import { useNavigate } from "react-router-dom";
+import mainImage from "../../assets/imgMainPage.jpg";
+import GreenBlobImage from "../../assets/blob_green.jpg";
+import YellowBlobImage from "../../assets/blob_yellow.jpg";
+import PurpleBlobImage from "../../assets/blob_dusty_purple.jpg";
+import items from "../../data/data"
+import ArrowUp from "../../assets/icons/icons-arrow-up.png";
+import ArrowSearch from "../../assets/icons/icon-search.svg";
 
-const HomePage = () => {
+const scrollToTop = () => {
+  window.scrollTo({
+    top: 0,
+    behavior: 'smooth',
+  });
+};
 
+const HomePage = () => {
   const navigate = useNavigate();
+
   const search = () => {
     navigate("/search");
   };
-  const login = () => {
-    navigate("/login");
-  };
-  const register = () => {
-    navigate("/register");
-  };
-
 
   return (
-    <>
-      <h1 className="text-3xl font-spartan font-bold">HOME PAGE</h1>
-      <button onClick={search}>Search classes</button>
-      <button onClick={login}>Log In</button>
-      <button className='bg-red hover:bg-pureWhite hover:text-red hover:border-2 hover:border-red text-white font-spartan font-semibold text-lg py-1 px-7 rounded-lg' 
-              onClick={register}>Join</button>
-    </>
+    <div className="flex flex-col my-auto lg:mx-auto lg:px-8 sm:mx-10 sm:my-12">
+      <section className="flex flex-col lg:flex-row lg:justify-between justify-center sm:pl-2">
+        <div className="lg:pt-36 pt-4 sm:pt-10 lg:text-left text-center items-center justify-center">
+          <h1 className="text-4xl sm:text-5xl md:text-6xl font-spartan font-semibold tracking-wide leading-tight pt-3 mb-8">
+            Helping unlock
+            <br />
```

Comment:
It's a good idea to apply flex properties to your text instead of using `br`. Using flexbox will work better with the responsiveness of your app. It'll also make it easier to align your text.